File: mobile/src/main/java/com/hackedbuce/aerialdream/AerialService.java
package com.hackedbuce.aerialdream;

import com.hackedbuce.aerialdream.data.Videos;

import java.util.List;

import retrofit.http.GET;
import retrofit.http.Path;
import rx.Observable;

/**
 * Created by rubix on 1/10/2016.
 */
public interface AerialService {

    @GET("{season}/videos/entries.json")
    Observable<List<Videos>> getVideos(@Path("season") String season);
}

File: mobile/src/main/java/com/hackedbuce/aerialdream/AerialService.kt
package com.hackedbuce.aerialdream

import com.hackedbuce.aerialdream.data.Video
import retrofit2.http.GET
import retrofit2.http.Path

interface AerialService {
    @GET("{season}/videos/entries.json")
    suspend fun getVideos(@Path("season") season: String): List<Video>
}
